pages/reference-files/[id].js

```diff
@@ -48,35 +55,52 @@ export default function ReferenceFile({
           <FileHeaderDownload file={referenceFile} />
         </ObjectPageHeader>
         <JsonDisplay item={referenceFile} isJsonFormat={isJson}>
-          <DataPanel>
-            <DataArea>
-              <FileDataItems
-                item={referenceFile}
-                fileSet={fileSet}
+          <FileDataItems item={referenceFile} fileSet={fileSet}></FileDataItems>
+          {("assembly" in referenceFile || "source_url" in referenceFile) && (
```

Comment:
Especially since you use the dot syntax a few lines down, you can use `referenceFile.assembly  || referenceFile.source_url`. This isn’t equivalent to using the `in` syntax, but they work for this case.

Reply:
Changed.